jenkins-inbound-agent/Dockerfile

```diff
@@ -61,5 +61,5 @@ RUN mkdir -p /home/jenkins/.ssh && \
 
 USER ${user}
 
-RUN tfenv install 0.11.7 && \
-  tfenv use 0.11.7
+RUN tfenv install 0.12.2 && \
```

Comment:
this is a breaking change, not sure if it's something we want to do? what's the driver for this?

Reply:
No rush. Let's postpone it